pkg/controller/certificaterequests/acme/acme_test.go

```diff
@@ -194,7 +194,7 @@ func TestSign(t *testing.T) {
 			builder: &testpkg.Builder{
 				CertManagerObjects: []runtime.Object{baseCR.DeepCopy(), baseIssuer.DeepCopy()},
 				ExpectedEvents: []string{
-					"Normal OrderCreated Created Order resource default-unit-test-ns/test-cr-3921610499",
+					"Normal OrderCreated Created Order resource default-unit-test-ns/test-cr-1733622556",
```

Comment:
These tests all use resource names derived from a hash of the json representation of the CR.
But because the CR fields have changed, the hash has changed.